src/main/java/step3/RandomMoveStrategy.java

```diff
@@ -0,0 +1,19 @@
+package step3;
+
+import java.util.Random;
+
+public class RandomMoveStrategy implements MoveStrategy {
+
+    @Override
+    public boolean move() {
+        if (getRandomValue() > 3) {
```

Comment:
제가 봤을땐, 3이라는 단어가 어떤걸 의미하길래 true인건지 알기가 어려운거 같아요.
상수화를 통해 나타내면 어떨까요?

아울러 already conditional을 사용하면 어떨까요?

```suggestion
        return getRandomValue() > 3
```

이런식으로 말이죠.